nodes/main.tf

```diff
@@ -22,13 +22,13 @@ resource "cloudca_volume" "volume" {
 
 resource "null_resource" "mount_volume" {
   count      = var.node_count
-  depends_on = ["cloudca_instance.node", "cloudca_volume.volume"]
+  depends_on = [cloudca_instance.node, cloudca_volume.volume]
 
   provisioner "remote-exec" {
     inline = [
       "if [ -d /data ]; then exit 0; fi",
       "sudo mkdir /data",
-      "sudo parted /dev/xvdb mklabel gpt mkpart ext4 0 100% i",
+      "sudo parted /dev/xvdb --script mklabel gpt mkpart ext4 0 100%",
```

Comment:
What does `--script` flag do here?

Reply:
It will prevent prompts, on Ubuntu it works with the "i" in the end but if you want to use it in any other OS it is best to use --script to prevent prompts.

Reply:
Oh I see, nice! :+1: